downloads/swan-lake-release-notes/swan-lake-beta3/RELEASE_NOTE.md

```diff
@@ -0,0 +1,796 @@
+
+```java
+    boolean isIsolated();
+
+    boolean isIsolated(String methodName);
+```
+
+#### Removed the Package Version from the Runtime
+
+The fully-qualified package version has been removed from the runtime and will only have the major version. Therefore, when you provide the version to the Ballerina runtime Java API like creating runtime values, you need to provide only the package runtime version. The stack traces will contain only the major package versions.
+
+
+### Bug Fixes
+
+The completion type of a stream is now considered in runtime assignability checks.
+
+```ballerina
+stream<int, error?> stm = new (new StreamImplementor());
+ 
+// Evaluated to true in SL Beta2, evaluates to false now.
+boolean streamCheck = stm is stream<int>;
+```
+
+To view bug fixes, see the [GitHub milestone for Swan Lake Beta3](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+is%3Aclosed+milestone%3A%22Ballerina+Swan+Lake+-+Beta3%22+label%3AType%2FBug+label%3ATeam%2FjBallerina).
+
+## Standard Library Updates
+
+### New Features
+
+#### `crypto` Package
+- Improved the hash APIs for cryptographic salt
+
+#### `graphql` Package
+- Added field alias support for GraphQL documents
+- Added variable support in GraphQL requests
+- Added mutation support for GraphQL services
+- Added typename introspection
+- Added input objects support
+- Added block string support
+- Added `graphql:Context` to support sharing meta-information between the resolvers
+
+#### `grpc` Package
+- Added declarative auth configurations
+- Added timestamp, duration, and struct type support
+- Added OAuth2 JWT bearer grant type support for the client
+- Introduced the support for a directory with PROTO files as the input (`--input` flag) for the gRPC command
+- Introduced the support for external import paths in the gRPC command using the `--proto_path` flag
+- Added the PROTO file name as a suffix for the `ROOT_DESCRIPTOR` constant and `getDescriptorMap` function to fix the re-declared symbol issue when multiple stub files are put into the same module. If you are going to regenerate the stub files with the Swan Lake Beta3 release, you need to change the service annotation like below.
+
+```ballerina
+@grpc:ServiceDescriptor {
+   descriptor: ROOT_DESCRIPTOR_<PROTO_FILE_NAME>,
+   descMap: getDescriptorMap<Proto_File_Name>()
+}
+service "HelloWorld" on new grpc:Listener(9090) {
+```
+
+#### `http` Package
+- Enabled HTTP trace and access log support
+- Added HATEOAS link support
+- Introduced the `http:Cache` annotation to the resource signature
+- Introduced support for the service-specific media-type subtype prefix
+- Introduced the introspection resource method to get the generated OpenAPI document of the service
+- Added OAuth2 JWT bearer grant type support for client
+
+#### `jwt` Package
+- Added HMAC signature support for JWT
+  
+#### `log` Package
+- Added observability span context values to the log messages when observability is enabled.
```

Comment:
Check the consistency of using fullstop again in all places.

Reply:
Fixed.